lib/rules/no-invalid-html-attribute.js

```diff
@@ -51,152 +57,160 @@ const VALID_VALUES = new Map([
   ['rel', rel],
 ]);
 
+/**
+ * Map between attributes and a mapping between pair-values and a set of values they are valid with
+ * @type {Map<string, Map<string, Set<string>>>}
+ */
+const VALID_PAIR_VALUES = new Map([
+  ['rel', pairs],
+]);
+
 /**
  * The set of all possible HTML elements. Used for skipping custom types
  * @type {Set<string>}
  */
 const HTML_ELEMENTS = new Set([
-  'html',
-  'base',
-  'head',
-  'link',
-  'meta',
-  'style',
-  'title',
-  'body',
+  'a',
+  'abbr',
+  'acronym',
   'address',
+  'applet',
+  'area',
   'article',
   'aside',
-  'footer',
```

Comment:
let's revert sorting this list so the diff is cleaner

Reply:
sure, this was just for me to get acquainted with the code. Sorting things help me anticipate what the next item in the list will be and allow me to spot missing values, e.g. when there was no `strong` after `strike` I'd directly see that it would be missing.
Reverted in 5d92d00

Reply:
Totally fine to do it in a separate PR, just makes this one harder to review :-)

Reply:
https://github.com/yannickcr/eslint-plugin-react/pull/3182 😉